app/react/UI/Icon/library.js

```diff
@@ -88,6 +88,7 @@ import { faUsers } from '@fortawesome/free-solid-svg-icons/faUsers';
 import { faUserTimes } from '@fortawesome/free-solid-svg-icons/faUserTimes';
 import { faHandPaper } from '@fortawesome/free-solid-svg-icons/faHandPaper';
 import { saveAndNext } from './save-and-next';
+import { exportCsv } from './export-csv';
```

Comment:
Was there really no icon in the entire font-awesome library that we had to create our own icon? Couldn't we have picked an icon from the library and added it here? Can you elaborate on how this happened? Thanks.

Reply:
It was requested by @simonfossom from his work in the new look and feel. There is a download/export icon that could be useful but the icon he proposed is a merge between a file download and a grid that better represent a CSV export. 
Maybe we can have a conversation about that if we prefer avoiding to create custom icons.